<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentGenerationDialog</class>
 <widget class="QDialog" name="DocumentGenerationDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vytvoření zpěvníku</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../res/resources.qrc">
    <normaloff>:/icons/16/Moleskine_16px.png</normaloff>:/icons/16/Moleskine_16px.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <widget class="QWidget" name="verticalWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="twLeftBottom_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabSongList_2">
         <attribute name="icon">
          <iconset resource="../../res/resources.qrc">
           <normaloff>:/icons/16/Settings_16px.png</normaloff>:/icons/16/Settings_16px.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Nastavení</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QGroupBox" name="gbToc">
            <property name="title">
             <string>Obsah</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="_2">
             <item row="0" column="3">
              <widget class="QSpinBox" name="sbTocColumns">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Columns_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Sloupců:</string>
               </property>
               <property name="buddy">
                <cstring>sbTocColumns</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>8</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="gbLyrics">
            <property name="title">
             <string>Slova</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="1" colspan="3">
              <widget class="QCheckBox" name="cbGenerateChords">
               <property name="text">
                <string>Vysázet akordy</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_9">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Guitar_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Sloupců:</string>
               </property>
               <property name="buddy">
                <cstring>sbLyricsColumns</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Columns_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>8</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3">
              <widget class="QSpinBox" name="sbLyricsColumns">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
               <property name="value">
                <number>2</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_14">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Term_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Sekce:</string>
               </property>
               <property name="buddy">
                <cstring>cmbLyricsMode</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="cmbLyricsMode">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <item>
                <property name="text">
                 <string>Tak, jak je to v textu</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dle pořadí snímků</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Dle pořadí snímků, bez duplikátů</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Papír</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_20">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Typography_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Velikost písma:</string>
               </property>
               <property name="buddy">
                <cstring>sbPageMargins</cstring>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_13">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/File Landscape_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QSpinBox" name="cbFontSize">
               <property name="suffix">
                <string> pt</string>
               </property>
               <property name="minimum">
                <number>4</number>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>12</number>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Down Right_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Split Files_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Velikost papíru:</string>
               </property>
               <property name="buddy">
                <cstring>cmbPageSize</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>16</width>
                 <height>8</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="cmbPageBreakMode">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="currentIndex">
                <number>3</number>
               </property>
               <property name="sizeAdjustPolicy">
                <enum>QComboBox::AdjustToContents</enum>
               </property>
               <item>
                <property name="text">
                 <string>Zalamovat i uprostřed písně</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Nezalamovat uprostřed písně</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Každou píseň do vlasntího sloupce</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Každou píseň na vlastní stranu</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_15">
               <property name="pixmap">
                <pixmap resource="../../res/resources.qrc">:/icons/16/Add White Space_16px.png</pixmap>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QComboBox" name="cmbPageSize"/>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Zalamování stránek:</string>
               </property>
               <property name="buddy">
                <cstring>cmbPageBreakMode</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Okraje:</string>
               </property>
               <property name="buddy">
                <cstring>sbPageMargins</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="3">
              <widget class="QSpinBox" name="sbPageMargins">
               <property name="suffix">
                <string> mm</string>
               </property>
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="value">
                <number>5</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1" colspan="3">
              <widget class="QCheckBox" name="cbLandscapeOrientation">
               <property name="text">
                <string>Na šířku</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_10">
              <property name="pixmap">
               <pixmap resource="../../res/resources.qrc">:/icons/16/Numbers_16px.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QCheckBox" name="cbNumberSongs">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Číslovat písně</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_17">
              <property name="pixmap">
               <pixmap resource="../../res/resources.qrc">:/icons/16/Open_16px.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QCheckBox" name="cbOpenWhenDone">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Po vytvoření otevřít PDF soubor</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap">
           <pixmap resource="../../res/resources.qrc">:/icons/16/Info_16px.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokud ovládáte HTML, CSS a JavaScript, můžete si šablonu přizpůsobit – nachází se v &lt;span style=&quot;font-family: Consolas;&quot;&gt;{složka programu}/etc/songbookTemplate.html&lt;/span&gt;.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="pixmap">
           <pixmap resource="../../res/resources.qrc">:/icons/16/Info_16px.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nově je možné &lt;span style=&quot; font-weight:700;&quot;&gt;exportovat do PDF a HTML.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnStorno">
          <property name="text">
           <string>Zavřít</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/resources.qrc">
            <normaloff>:/icons/16/Delete_16px.png</normaloff>:/icons/16/Delete_16px.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnGenerate">
          <property name="text">
           <string>Vytvořit</string>
          </property>
          <property name="icon">
           <iconset resource="../../res/resources.qrc">
            <normaloff>:/icons/16/Save_16px.png</normaloff>:/icons/16/Save_16px.png</iconset>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="MultiSongSelectionWidget" name="wgtSongSelection" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiSongSelectionWidget</class>
   <extends>QWidget</extends>
   <header>wgt/multisongselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>wgtSongSelection</tabstop>
  <tabstop>twLeftBottom_2</tabstop>
  <tabstop>gbToc</tabstop>
  <tabstop>sbTocColumns</tabstop>
  <tabstop>gbLyrics</tabstop>
  <tabstop>sbLyricsColumns</tabstop>
  <tabstop>cmbLyricsMode</tabstop>
  <tabstop>cbGenerateChords</tabstop>
  <tabstop>cmbPageBreakMode</tabstop>
  <tabstop>cmbPageSize</tabstop>
  <tabstop>sbPageMargins</tabstop>
  <tabstop>cbNumberSongs</tabstop>
  <tabstop>cbOpenWhenDone</tabstop>
  <tabstop>btnGenerate</tabstop>
  <tabstop>btnStorno</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/resources.qrc"/>
 </resources>
 <connections/>
</ui>
